README.md

```diff
@@ -69,6 +70,22 @@ myComponentMetadata.addSim({
         items:['🧒 ', '👶 ', '🐊 ']
     }
 });
+
+// If you want to use the "snap" tool, this method must be called. Currently, the snap
+// tool relies on Stylable
+myComponentMetadata.exportedFrom({
+  path: 'src/my-comp/my-comp',                          // the path to your component, relative to the root, and without file extension
+  importName: 'MyComp',                                 // the name under which you export your component
+  baseStylePath: 'src/my-comp/my-comp/my-comp.st.css',  // the path to the base stylesheet for the component (as opposed to themes)
+  displayName: 'Composite'                              // the name you want to refer to the component with. Does not need to match importName
```

Comment:
I'm wondering why it would different from `importName` or `displayName` defined in the component file.